app/services/katello/registration_manager.rb

```diff
@@ -264,12 +264,20 @@ def set_host_collections(host, activation_keys)
         host.host_collection_ids = host_collection_ids
       end
 
-      def lookup_content_view_environment(activation_keys)
+      def lookup_content_view_environments(activation_keys)
+        if Setting['allow_multiple_content_views']
```

Comment:
Would be good to add comments clarifying:
If the setting is on, we combine all CVEs from all AKs
If the setting is off, we stick with the previous behavior (the last AK with a valid cv/lce wins).

Reply:
updated